packages/grid/_modules_/grid/components/default-footer.tsx

```diff
@@ -25,23 +25,20 @@ export const DefaultFooter = React.forwardRef<HTMLDivElement, DefaultFooterProps
       return null;
     }
 
-    const isPaginationAvailable = !!paginationComponent;
-    const showRowCount = !options.hideFooterRowCount && !isPaginationAvailable && (
-      <RowCount rowCount={totalRowCount} />
-    );
-    const showSelectedRowCount = !options.hideFooterSelectedRowCount && (
-      <SelectedRowCount selectedRowCount={selectedRowCount} />
-    );
-    const justifyItemsEnd = !selectedRowCount && !options.hideFooterSelectedRowCount;
+    const showSelectedRowCount =
+      !options.hideFooterSelectedRowCount && selectedRowCount > 0 ? (
+        <SelectedRowCount selectedRowCount={selectedRowCount} />
+      ) : (
+        <div />
```

Comment:
why not null?

Reply:
Needed to keep the CSS simple (flexbox space-between)